docs/micros/visor.md

```diff
@@ -1,8 +1,52 @@
-# Visor
-You can view recent logs on visor.
+---
+id: visor
+title: Using Visor
+sidebar_label: Visor
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+### Visor
+
+Something not working? Use Visor to see a live view of the result of every request processed! Using Visor, you can also test your endpoints by sending your Micro requests!
 
 ### Opening Visor
-Head over to your Deta console to open your micro project; then open the *visor* tab.
+<Tabs 
+    defaultValue="browser" 
+    values={[
+    { label: 'Browser', value: 'browser' },
+    { label: 'Terminal', value: 'terminal' },
+    ]
+}>
+<TabItem value="browser">
+
+Start off by logging into your console at [deta.sh](https://deta.sh).
+
+Once that's done, choose a micro to view by clicking on itname (under the _"Micros"_ tab).
+
+Now click on _"Visor"_
+
+</TabItem>
+<TabItem value="terminal">
+
+Start your terminal in the root directory of your deployed Detproject.
+
+Run `deta visor open`.
+
+</TabItem>
+</Tabs>
+
+### Using Visor
+When viewing the Visor page, you'll see a link to your Micro in the top right, a button named _"HTTP Client"_ in the bottom left and a list of handled events by your Micro!
+
+#### The Event List
+Every event has the returned status code, the path, the protocol, the method and the time logged above it. Under this information are three  tabs, letting you view the response given by the Micro, the Request sent to the Micro and logs (normally displaying errors if they were to occur).

```

Comment:
We actually have four tabs, 'Errors' being the last one. 

I personally think playing around with all the edge cases & documenting the product with visuals in the process would be a great addition here!

Reply:
We do? :D

![grafik](https://user-images.githubusercontent.com/14904204/115006019-af705100-9ea8-11eb-9b79-fe8652b89838.png)


Good idea on the visuals and edge case idea, I'll make sure to work those in!